fml/message_loop_task_queues.h

```diff
@@ -119,14 +120,14 @@ class MessageLoopTaskQueues
   bool Merge(TaskQueueId owner, TaskQueueId subsumed);
```

Comment:
Could you please update the comment above this method to take into account that a queue can subsume multiple task queue ids?